### PR TITLE
FLINT 2.8.3 for Julia 1.0-1.5

### PR DESCRIPTION
**Pull request by fingolfin**
CC @thofma @tthsqe12 @wbhart

Once this is merged, we can make another PR with `build_for_julia16_or_newer = true` to cover Julia >= 1.6. And of course update `LoadFlint.jl` *sigh*

### CODE REVIEW

**Comment by thofma** [comment]
I will need to release the new LoadFlint ASAP once this here is merged. So please don't merge in the middle of the night.

**Review by thofma — APPROVED** [review]

**Comment by fingolfin** [comment]
May I suggest this:
- we (well, @giordano :-) ) merge this
- then we stop the automerge to the general registry
- then LoadFlint.jl is adapted whenever possible
- finally the merge of the JLL can be resumed

One thing I am not 100% sure is how to fit in the second update for the JLL (for Julia >= 1.6). Perhaps it suffices to do this last, after LoadFlint is updated and the first JLL update is merged? I think it'd work as users on Julia >= 1.6 would simply get the new LoadFlint but it would simply load the older FLINT_jll until we update...

**Comment by thofma** [comment]
Agreed, this is a good plan.

**Pull request merged**
